### DISCUSSION TITLE
how do i make a page including dot in nuxt?

### DISCUSSION THREAD

**Discussion by slim-python** [vercel/next.js#29045] [Help]
every time i make a file name `robots.txt.vue` in order to get  `localhost/robots.txt` i get an error saying this page can't be found
but when i make a file name `robots_txt.vue` it works and i get `localhost/robots_txt`

how do I get the previous result?
I want to have like this:  **example.com/robots.txt**

![image](https://user-images.githubusercontent.com/55599278/133019451-e9f82f2b-af0a-4d80-a404-429b1be0cf27.png)

**Comment by slim-python** [comment]
solved it
https://nuxtjs.org/docs/2.x/directory-structure/static

**Comment by wilsonneto-swe** [comment]
@slim-python Nuxt and Next are different things and maintained by different teams and repos.

Nuxt: https://github.com/nuxt
